src/containers/layouts/Scaffold/styles.js

```diff
@@ -14,11 +14,11 @@ const useStyles = makeStyles((theme) => ({
     },
     "& ::-webkit-scrollbar-track": {
       marginTop: theme.spacing(1),
-      webkitBoxShadow: `inset 0 0 8px ${theme.palette.common.black}`,
-      backgroundColor: theme.palette.border.main,
+      webkitBoxShadow: `inset 0 0 8px ${theme?.palette?.common?.black}`,
```

Comment:
Why are we using `?` with theme, it should always be there right? Why are we checking if it is undefined?